.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -0,0 +1,128 @@
+name: Bug Report
+description: 'You have noticed a general issue or regression, and would like to report it'
+projects:
+  - jellyfin/jellyfin-web
```

Comment:
```suggestion
```
won't work for users without write access to the project in question, at least according to the docs

Reply:
Leaving for now, as this might change in future